_posts/2020-07-08-Book-Five-Dysfunctions.md

```diff
@@ -0,0 +1,173 @@
+---
```

Comment:
General remark : I would also have explicitly mentioned the 5 side -effects of the dysfunctions :
Invulnerability, ARtificial Harmony, Ambiguity, Low standards & Status & ego